s-ramp-ide/srampRepositoryBrowser/contexts.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<contexts>
+	<context id="cz.muni.fi.srampRepositoryBrowser.views.RepositoryBrowser" >
+		<description>S-RAMP repository browser provides view for using S-RAMP repository.</description>
```

Comment:
"The S-RAMP Repository Browser provides search and browsing within the S-RAMP repository, as well as a means to download and re-use artifacts stored within it."